WhoisthatWebAPI/Controllers/FacesController.cs

```diff
@@ -1,12 +1,12 @@
-﻿using System.Collections.Generic;
+using System.Collections.Generic;
 using System.Linq;
 using Microsoft.AspNetCore.Mvc;
 using WhoisthatCommon;
 using WhoisthatWebAPI.RequestObjects;
 
 namespace WhoisthatWebAPI.Controllers
 {
-    [Route("api/[controller]")]
+    [Route("api/faces")]
```

Comment:
This line should not be touched
```suggestion
    [Route("api/[controller]")]
```

Reply:
[controller] was good enough